AutoT4MVC/AutoT4MVC.csproj

```diff
@@ -38,7 +39,7 @@
     <AssemblyName>AutoT4MVC</AssemblyName>
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>Key.snk</AssemblyOriginatorKeyFile>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
Was there a reason for this change? Was the 4.6 upgrade necessary. I'm concerned it could cause problems for people still using older versions of VS.